python pratices/assignment.10.txt

```diff
@@ -0,0 +1,16 @@
+def Remove(duplicate): 
+    
+final_list = [] 
```

Comment:
Where is Indentation. After defining a function, whatever statements you want to execute in that function, you have to give a indentation of  4 spaces or 1 tab before the statement